bindings/python/crocoddyl/core/activations/quadratic-flat.cpp

```diff
@@ -0,0 +1,45 @@
+///////////////////////////////////////////////////////////////////////////////
+// BSD 3-Clause License
+//
+// Copyright (C) 2018-2020, LAAS-CNRS, University of Edinburgh
+// Copyright note valid unless otherwise stated in individual files.
+// All rights reserved.
+///////////////////////////////////////////////////////////////////////////////
+
+#include "python/crocoddyl/core/core.hpp"
+#include "python/crocoddyl/core/activation-base.hpp"
+#include "crocoddyl/core/activations/quadratic-flat.hpp"
+
+namespace crocoddyl {
+namespace python {
+
+void exposeActivationQuadFlat() {
+  bp::class_<ActivationModelQuadFlat, bp::bases<ActivationModelAbstract> >(
+      "ActivationModelQuadFlat",
+      "Quadratic flat activation model.\n\n"
+      "A quadratic flat action describes a quadratic flat function that depends on the residual, i.e.\n"
+      "1 - exp(*||r||^2 / sigma2).",
+      bp::init<int,double>(bp::args("self", "nr","sigma2"),
```

Comment:
Why calling it as sigma2? or better why do we call it as sigma?
(same for the other activation)

Reply:
No real reason, I changed it for alpha